src/actuator/water_pump.py

```diff
@@ -0,0 +1,44 @@
+from tools.status import Status
+from actuator.actuator import Actuator
+from pubsub import pub
+import RPi.GPIO as GPIO
+
+# Actuator Interface
+
+
+class Waterpump(Actuator):
+    PUMP_PIN = 10
+
+    def __init__(self, *args, **kwargs):
+
+        """__init__ Initialise an Actuator Interface.
+        :param actuator_type:
+        :type actuator_type: str
+        :param actuator_id:
+        :type actuator_id: int
+        :param actuator_status:
+        :type actuator_status: Status
+        """
+        super().__init__("water_pump", args, kwargs)
+        self._on = 0
```

Comment:
Convert to boolean if its a flag, maybe rename to `_is_on`
```suggestion
        self._on = True
```